numba/testing/__init__.py

```diff
@@ -34,7 +34,7 @@ def load_testsuite(loader, dir):
 
 
 def run_tests(argv=None, defaultTest=None, topleveldir=None,
-              xmloutput=None, verbosity=1, nomultiproc=False):
+              xmloutput="junit_reports", verbosity=1, nomultiproc=False):
```

Comment:
Let's change the argparser so that xmloutput is read from the cmdline. Keep the default to not producing xml.